### PR TITLE
Luca-AlexanerThomas.AutoHome-Autostart version 1.0.0

### PR DESCRIPTION
**Pull request by luca-alexander-thomas**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23633)

### CODE REVIEW

File: manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+PackageIdentifier: Luca-AlexanerThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+PackageLocale: de-DE
+Publisher: Luca-Alexander Thomas
+PublisherUrl: https://lucathomas.de
+PublisherSupportUrl: https://lucathomas.de/kontakt
+PrivacyUrl: https://lucathomas.de/datenschutzerklaerung/
+Author: Luca-Alexander Thomas
+PackageName: AutoHome-Autostart
```
Comment (OfficialEsco) [inline manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml:12]: This needs to match **A**dd or **R**emove **P**rograms (ARP)
```suggestion
PackageName: AutoHome
```

Comment (OfficialEsco) [inline manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml:12]: This needs to match **A**dd or **R**emove **P**rograms (ARP)
```suggestion
PackageName: AutoHome
```

**Comment by luca-alexander-thomas** [comment]
@msftbot

File: manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+PackageUrl: https://github.com/luca-alexander-thomas/AutoHome
+License: GPL-3.0
+LicenseUrl: https://github.com/luca-alexander-thomas/AutoHome/blob/Autostart/LICENSE
+Copyright: 2021 by Luca-Alexander Thomas
+ShortDescription: AutoHome ist ein Autostart Programm welches Programme Startet wenn ein bestimmtes Laufwerk vorhanden ist. AutoHome ist Perfekt geeignet für Dock Setups Zuhause oder auf der Arbeit.
+Description: AutoHome ist ein Autostart Programm welches Programme Startet wenn ein bestimmtes Laufwerk vorhanden ist. AutoHome ist Perfekt geeignet für Dock Setups Zuhause oder auf der Arbeit.
+Moniker: AutoHome
+Tags:
+- Autostart
```
Comment (OfficialEsco) [inline manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml:21]: These should be lower case for consistency
```suggestion
Moniker: autohome
Tags:
- autostart
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/23633#pullrequestreview-721610061

**Comment by luca-alexander-thomas** [comment]
Hello What i have now to do....

**Pull request closed without merging**